ethereum-common/src/calls.rs

```diff
@@ -1,14 +1,21 @@
 use crate::pb::sf::substreams::ethereum::v1::{Call, Calls};
-use substreams::pb::sf::substreams::index::v1::Keys;
 use crate::pb::sf::substreams::v1::Clock;
 use anyhow::Ok;
 use substreams::errors::Error;
 use substreams::matches_keys_in_parsed_expr;
+use substreams::pb::sf::substreams::index::v1::Keys;
 use substreams::Hex;
 use substreams_ethereum::pb::eth::v2::Block;
 
 #[substreams::handlers::map]
 fn all_calls(blk: Block) -> Result<Calls, Error> {
+    _all_calls(blk)
+}
+
+/*
+    all_calls is equal to _all_calls. This is just for unit testing purposes.
+*/
```

Comment:
Let's use standard Rust comments and fix the description which seems inversed.

```suggestion
/// _all_calls is equal to [all_calls] but exists only for unit testing purposes.
```